src/Core/Services/MetadataProviders/SqlMetadataProvider.cs

```diff
@@ -469,8 +470,12 @@ protected virtual async Task FillSchemaForStoredProcedureAsync(
                     }
                     else
                     {
-                        parameterDefinition.HasConfigDefault = true;
-                        parameterDefinition.ConfigDefaultValue = configParamValue?.ToString();
+                        // Map all metadata from config
```

Comment:
Eventually, we should be augmenting this metadata in SQL. So, that config has less information and we expose that schema info from the db directly as a tool to the MCP client.